### PR TITLE
Empty state: Add animation for Grot

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- some friday fun 🥳 
- creates a new `GrotNotFound` component
- adds a minor cursor tracking animation
  - have throttled this not to cause any huge performance issues
  - imo this is a good place to put animations since it's going to be used in an empty state where there's not much else going on
- uses the new `GrotNotFound` component in the command palette empty state
- uses the new `GrotNotFound` component in the page 404 state

# animation
https://github.com/grafana/grafana/assets/20999846/cc413199-9650-46ec-86fb-c408bce23d6f

# animation when cpu throttling 6x
https://github.com/grafana/grafana/assets/20999846/5ff7dd28-5056-445d-8e54-006d3119f351

# animation on 404 page
https://github.com/grafana/grafana/assets/20999846/ad33121d-3caa-4b88-bfea-9a65b044268d

**Why do we need this feature?**

- make grot more interactive! 

**Who is this feature for?**

- everyone

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/core/components/GrotNotFound/GrotNotFound.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  return <SVG src={notFoundSvg} className={styles.svg} height={height} width={width} />;
+};
+
+GrotNotFound.displayName = 'GrotNotFound';
+
+const getStyles = (theme: GrafanaTheme2, xPos: number | null, yPos: number | null) => {
+  const { innerWidth, innerHeight } = window;
+  const heightRatio = yPos && yPos / innerHeight;
+  const widthRatio = xPos && xPos / innerWidth;
+  const rotation = heightRatio !== null ? getIntermediateValue(heightRatio, -20, 5) : 0;
```
Comment (tskarhed) [inline public/app/core/components/GrotNotFound/GrotNotFound.tsx:29]: I guess you eyeballed the start and end values?

**Review by tskarhed — APPROVED** [review]
YES!

File: public/app/core/components/GrotNotFound/GrotNotFound.tsx (hunk 1 above)
Comment (ashharrison90) [reply public/app/core/components/GrotNotFound/GrotNotFound.tsx]: yup, i've extracted them out into constants now as well 👍

**Review by eledobleefe — APPROVED** [review]
LGTM 🎉

**Comment by torkelo** [comment]
@ashharrison90 love this! ❤️

**Pull request merged**